polkadot/primitives/src/v7/mod.rs

```diff
@@ -1087,9 +1087,12 @@ pub enum CoreState<H = Hash, N = BlockNumber> {
 
 impl<N> CoreState<N> {
 	/// If this core state has a `para_id`, return it.
```

Comment:
Let's align the doc comment with the PRDoc, and state that it always returns the scheduled `ParaId` if any, and otherwise None.

Reply:
Sure, forgot about the doc comment.